src/concrete/ml/quantization/quantized_module.py

```diff
@@ -90,13 +92,16 @@ class QuantizedModule:
     input_quantizers: List[UniformQuantizer]
     output_quantizers: List[UniformQuantizer]
     fhe_circuit: Union[None, Circuit]
+    _onnx_preprocessing: Optional[onnx.ModelProto]
```

Comment:
here here are defined attributes that are not init input parameters, so better to remove this `_onnx_preprocessing: Optional[onnx.ModelProto]` but add typing to `self._onnx_preprocessing = onnx_preprocessing` below

Reply:
What about all the over attributes above? Some of them appear in the init. But yes I can move it.